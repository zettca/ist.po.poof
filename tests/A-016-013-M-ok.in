4
solo
5
4
..
4
vader
4
private
3
pu
1
0
0
