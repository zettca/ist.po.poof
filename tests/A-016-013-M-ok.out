Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Identificador do utilizador: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Indique o nome do directório: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Indique o nome do directório: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Indique o nome do directório: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Indique o nome da entrada: Remover entrada: Operação inválida: O utilizador 'solo' não tem permissão para realizar a operação.
Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: d - vader 32 .
d - vader 32 ..
d w yoda 16 pu
d w vader 16 pvader
Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 